<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>test discover</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
<style type="text/css" media="screen">
body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
table       { font-size: 100%; }
pre         { }

/* -- heading ---------------------------------------------------------------------- */
h1 {
	font-size: 16pt;
	color: gray;
}
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .attribute {
    margin-top: 1ex;
    margin-bottom: 0;
}

.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}

/* -- css div popup ------------------------------------------------------------------------ */
a.popup_link {
}

a.popup_link:hover {
    color: red;
}

.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #E6E6D6;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 8pt;
    width: 500px;
}

}
/* -- report ------------------------------------------------------------------------ */
#show_detail_line {
    margin-top: 3ex;
    margin-bottom: 1ex;
}
#result_table {
    width: 80%;
    border-collapse: collapse;
    border: 1px solid #777;
}
#header_row {
    font-weight: bold;
    color: white;
    background-color: #777;
}
#result_table td {
    border: 1px solid #777;
    padding: 2px;
}
#total_row  { font-weight: bold; }
.passClass  { background-color: #6c6; }
.failClass  { background-color: #c60; }
.errorClass { background-color: #c00; }
.passCase   { color: #6c6; }
.failCase   { color: #c60; font-weight: bold; }
.errorCase  { color: #c00; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }


/* -- ending ---------------------------------------------------------------------- */
#ending {
}

</style>

</head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* level - 0:Summary; 1:Failed; 2:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level < 1) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level > 1) {
                tr.className = '';
            }
            else {
                tr.className = 'hiddenRow';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}

/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>

<div class='heading'>
<h1>test discover</h1>
<p class='attribute'><strong>Start Time:</strong> 2019-02-27 09:32:12</p>
<p class='attribute'><strong>Duration:</strong> 0:00:03.728407</p>
<p class='attribute'><strong>Status:</strong> Pass 1 Failure 10 Error 26</p>

<p class='description'></p>
</div>



<p id='show_detail_line'>Show
<a href='javascript:showCase(0)'>Summary</a>
<a href='javascript:showCase(1)'>Failed</a>
<a href='javascript:showCase(2)'>All</a>
</p>
<table id='result_table'>
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row'>
    <td>Test Group/Test case</td>
    <td>Count</td>
    <td>Pass</td>
    <td>Fail</td>
    <td>Error</td>
    <td>View</td>
</tr>

<tr class='errorClass'>
    <td>test_Training_resources.Trainning_resourses</td>
    <td>6</td>
    <td>1</td>
    <td>2</td>
    <td>3</td>
    <td><a href="javascript:showClassDetail('c1',6)">Detail</a></td>
</tr>

<tr id='ft1.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_Trainning_resourses_add</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
        error</a>

    <div id='div_ft1.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 77, in test_Trainning_resourses_add
    resp = self.start.judge_method(method, url, data, headers=None)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 77, in test_Trainning_resourses_add
    resp = self.start.judge_method(method, url, data, headers=None)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='pt1.2' class='hiddenRow'>
    <td class='none'><div class='testcase'>test_Trainning_resourses_change</div></td>
    <td colspan='5' align='center'>pass</td>
</tr>

<tr id='ft1.3' class='none'>
    <td class='errorCase'><div class='testcase'>test_Trainning_resourses_givePeople</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.3')" >
        error</a>

    <div id='div_ft1.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.3: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 161, in test_Trainning_resourses_givePeople
    resp = self.start.judge_method(method, url, data, headers=None)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 161, in test_Trainning_resourses_givePeople
    resp = self.start.judge_method(method, url, data, headers=None)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.4' class='none'>
    <td class='failCase'><div class='testcase'>test_Trainning_resourses_public</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.4')" >
        fail</a>

    <div id='div_ft1.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.4: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 146, in test_Trainning_resourses_public
    raise AssertionError(errors02)
AssertionError: ["request() got an unexpected keyword argument 'file'", "request() got an unexpected keyword argument 'file'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 146, in test_Trainning_resourses_public
    raise AssertionError(errors02)
AssertionError: ["request() got an unexpected keyword argument 'file'", "request() got an unexpected keyword argument 'file'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.5' class='none'>
    <td class='failCase'><div class='testcase'>test_Trainning_resourses_query</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.5')" >
        fail</a>

    <div id='div_ft1.5' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.5').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.5: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 61, in test_Trainning_resourses_query
    raise AssertionError(errors01)
AssertionError: ["request() got an unexpected keyword argument 'file'", "request() got an unexpected keyword argument 'file'", "request() got an unexpected keyword argument 'file'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 61, in test_Trainning_resourses_query
    raise AssertionError(errors01)
AssertionError: ["request() got an unexpected keyword argument 'file'", "request() got an unexpected keyword argument 'file'", "request() got an unexpected keyword argument 'file'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.6' class='none'>
    <td class='errorCase'><div class='testcase'>test_Trainning_resourses_throw</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.6')" >
        error</a>

    <div id='div_ft1.6' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.6').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.6: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 95, in test_Trainning_resourses_throw
    resp = self.start.judge_method(method, url, data, headers=None)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_Training_resources.py", line 95, in test_Trainning_resourses_throw
    resp = self.start.judge_method(method, url, data, headers=None)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='failClass'>
    <td>test_common_resource_pool.decode</td>
    <td>1</td>
    <td>0</td>
    <td>1</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c2',1)">Detail</a></td>
</tr>

<tr id='ft2.1' class='none'>
    <td class='failCase'><div class='testcase'>test_common_resource_pool</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.1')" >
        fail</a>

    <div id='div_ft2.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_common_resource_pool.py", line 39, in test_common_resource_pool
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_common_resource_pool.py", line 46, in test_common_resource_pool
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_common_resource_pool.py", line 39, in test_common_resource_pool
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_common_resource_pool.py", line 46, in test_common_resource_pool
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='failClass'>
    <td>test_login.login_test</td>
    <td>2</td>
    <td>0</td>
    <td>2</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c3',2)">Detail</a></td>
</tr>

<tr id='ft3.1' class='none'>
    <td class='failCase'><div class='testcase'>test_logoin_06</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.1')" >
        fail</a>

    <div id='div_ft3.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft3.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft3.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_login.py", line 93, in test_logoin_06
    raise AssertionError(error)
AssertionError: ["第3条测试用例执行失败，失败信息是'errors' != 'error'\n- errors\n?      -\n+ error\n", "第4条测试用例执行失败，失败信息是'checkcodeErrors' != 'checkcodeError'\n- checkcodeErrors\n?               -\n+ checkcodeError\n", "第8条测试用例执行失败，失败信息是'successful' != 'error'\n- successful\n+ error\n", "第9条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n", "第10条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_login.py", line 93, in test_logoin_06
    raise AssertionError(error)
AssertionError: ["第3条测试用例执行失败，失败信息是'errors' != 'error'\n- errors\n?      -\n+ error\n", "第4条测试用例执行失败，失败信息是'checkcodeErrors' != 'checkcodeError'\n- checkcodeErrors\n?               -\n+ checkcodeError\n", "第8条测试用例执行失败，失败信息是'successful' != 'error'\n- successful\n+ error\n", "第9条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n", "第10条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft3.2' class='none'>
    <td class='failCase'><div class='testcase'>test_logoin_07</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.2')" >
        fail</a>

    <div id='div_ft3.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft3.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft3.2: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_login.py", line 115, in test_logoin_07
    raise AssertionError(error)
AssertionError: ["第3条测试用例执行失败，失败信息是'errors' != 'error'\n- errors\n?      -\n+ error\n\n", "第4条测试用例执行失败，失败信息是'checkcodeErrors' != 'checkcodeError'\n- checkcodeErrors\n?               -\n+ checkcodeError\n\n", "第8条测试用例执行失败，失败信息是'successful' != 'error'\n- successful\n+ error\n\n", "第9条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n\n", "第10条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n\n"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_login.py", line 115, in test_logoin_07
    raise AssertionError(error)
AssertionError: ["第3条测试用例执行失败，失败信息是'errors' != 'error'\n- errors\n?      -\n+ error\n\n", "第4条测试用例执行失败，失败信息是'checkcodeErrors' != 'checkcodeError'\n- checkcodeErrors\n?               -\n+ checkcodeError\n\n", "第8条测试用例执行失败，失败信息是'successful' != 'error'\n- successful\n+ error\n\n", "第9条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n\n", "第10条测试用例执行失败，失败信息是'successful' != 'success'\n- successful\n?        ---\n+ success\n\n"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>test_student_01.student</td>
    <td>15</td>
    <td>0</td>
    <td>0</td>
    <td>15</td>
    <td><a href="javascript:showClassDetail('c4',15)">Detail</a></td>
</tr>

<tr id='ft4.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_01</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.1')" >
        error</a>

    <div id='div_ft4.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 68, in test_01
    self.assertEqual(response.text,'checkcodeError')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 68, in test_01
    self.assertEqual(response.text,'checkcodeError')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_02</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.2')" >
        error</a>

    <div id='div_ft4.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.2: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 78, in test_02
    self.assertEqual(response.text, 'yes')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 78, in test_02
    self.assertEqual(response.text, 'yes')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.3' class='none'>
    <td class='errorCase'><div class='testcase'>test_03</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.3')" >
        error</a>

    <div id='div_ft4.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.3: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 88, in test_03
    self.assertEqual(response.text, '二级密码输入错误')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 88, in test_03
    self.assertEqual(response.text, '二级密码输入错误')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.4' class='none'>
    <td class='errorCase'><div class='testcase'>test_04</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.4')" >
        error</a>

    <div id='div_ft4.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.4: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 99, in test_04
    self.assertEquals(response.json()['education'], '大专')
AttributeError: 'NoneType' object has no attribute 'json'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 99, in test_04
    self.assertEquals(response.json()['education'], '大专')
AttributeError: 'NoneType' object has no attribute 'json'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.5' class='none'>
    <td class='errorCase'><div class='testcase'>test_05</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.5')" >
        error</a>

    <div id='div_ft4.5' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.5').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.5: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 109, in test_05
    self.assertEqual(response.text, '')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 109, in test_05
    self.assertEqual(response.text, '')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.6' class='none'>
    <td class='errorCase'><div class='testcase'>test_06</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.6')" >
        error</a>

    <div id='div_ft4.6' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.6').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.6: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 121, in test_06
    self.assertEqual(response.text, '')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 121, in test_06
    self.assertEqual(response.text, '')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.7' class='none'>
    <td class='errorCase'><div class='testcase'>test_07</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.7')" >
        error</a>

    <div id='div_ft4.7' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.7').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.7: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 132, in test_07
    self.assertEqual(response.json()['totalRow'], 158)
AttributeError: 'NoneType' object has no attribute 'json'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 132, in test_07
    self.assertEqual(response.json()['totalRow'], 158)
AttributeError: 'NoneType' object has no attribute 'json'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.8' class='none'>
    <td class='errorCase'><div class='testcase'>test_08</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.8')" >
        error</a>

    <div id='div_ft4.8' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.8').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.8: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 144, in test_08
    self.assertEqual(response.text, '')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 144, in test_08
    self.assertEqual(response.text, '')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.9' class='none'>
    <td class='errorCase'><div class='testcase'>test_09</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.9')" >
        error</a>

    <div id='div_ft4.9' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.9').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.9: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 155, in test_09
    self.assertEqual(response.json()['totalRow'], 0)
AttributeError: 'NoneType' object has no attribute 'json'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 155, in test_09
    self.assertEqual(response.json()['totalRow'], 0)
AttributeError: 'NoneType' object has no attribute 'json'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.10' class='none'>
    <td class='errorCase'><div class='testcase'>test_10</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.10')" >
        error</a>

    <div id='div_ft4.10' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.10').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.10: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 162, in test_10
    parameter1 = self.readfile.dispose_params1(parameter)
  File "C:\woniuboss\common\Utility.py", line 27, in dispose_params1
    key,value=line.strip().split('=')
ValueError: not enough values to unpack (expected 2, got 1)
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 162, in test_10
    parameter1 = self.readfile.dispose_params1(parameter)
  File "C:\woniuboss\common\Utility.py", line 27, in dispose_params1
    key,value=line.strip().split('=')
ValueError: not enough values to unpack (expected 2, got 1)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.11' class='none'>
    <td class='errorCase'><div class='testcase'>test_11</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.11')" >
        error</a>

    <div id='div_ft4.11' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.11').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.11: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 177, in test_11
    self.assertEqual(response.text, 'success')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 177, in test_11
    self.assertEqual(response.text, 'success')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.12' class='none'>
    <td class='errorCase'><div class='testcase'>test_12</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.12')" >
        error</a>

    <div id='div_ft4.12' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.12').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.12: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 189, in test_12
    self.assertEqual(response.text, 'success')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 189, in test_12
    self.assertEqual(response.text, 'success')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.13' class='none'>
    <td class='errorCase'><div class='testcase'>test_13</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.13')" >
        error</a>

    <div id='div_ft4.13' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.13').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.13: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 203, in test_13
    self.assertEqual(response.text, 'success')
AttributeError: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 203, in test_13
    self.assertEqual(response.text, 'success')
AttributeError: 'NoneType' object has no attribute 'text'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.14' class='none'>
    <td class='errorCase'><div class='testcase'>test_14</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.14')" >
        error</a>

    <div id='div_ft4.14' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.14').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.14: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 214, in test_14
    self.assertEqual(response.json()['totalRow'], 158)
AttributeError: 'NoneType' object has no attribute 'json'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 214, in test_14
    self.assertEqual(response.json()['totalRow'], 158)
AttributeError: 'NoneType' object has no attribute 'json'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.15' class='none'>
    <td class='errorCase'><div class='testcase'>test_15</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.15')" >
        error</a>

    <div id='div_ft4.15' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.15').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.15: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 222, in test_15
    self.assertEqual(response.json()['totalRow'], 4)
AttributeError: 'NoneType' object has no attribute 'json'
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_01.py", line 222, in test_15
    self.assertEqual(response.json()['totalRow'], 4)
AttributeError: 'NoneType' object has no attribute 'json'


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>test_student_mode.Student_02</td>
    <td>8</td>
    <td>0</td>
    <td>0</td>
    <td>8</td>
    <td><a href="javascript:showClassDetail('c5',8)">Detail</a></td>
</tr>

<tr id='ft5.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_01</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.1')" >
        error</a>

    <div id='div_ft5.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 143, in test_stu_01
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 143, in test_stu_01
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_02</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.2')" >
        error</a>

    <div id='div_ft5.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.2: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 156, in test_stu_02
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 156, in test_stu_02
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.3' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_03</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.3')" >
        error</a>

    <div id='div_ft5.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.3: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 170, in test_stu_03
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 170, in test_stu_03
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.4' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_04</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.4')" >
        error</a>

    <div id='div_ft5.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.4: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 185, in test_stu_04
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 185, in test_stu_04
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.5' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_05</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.5')" >
        error</a>

    <div id='div_ft5.5' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.5').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.5: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 200, in test_stu_05
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 200, in test_stu_05
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.6' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_06</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.6')" >
        error</a>

    <div id='div_ft5.6' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.6').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.6: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 217, in test_stu_06
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 217, in test_stu_06
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.7' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_07</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.7')" >
        error</a>

    <div id='div_ft5.7' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.7').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.7: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 229, in test_stu_07
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 229, in test_stu_07
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.8' class='none'>
    <td class='errorCase'><div class='testcase'>test_stu_08</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.8')" >
        error</a>

    <div id='div_ft5.8' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.8').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.8: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 243, in test_stu_08
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 94, in student
    resp = self.con.judge_method(url=url,method=method,data=data,headers=header)
  File "C:\woniuboss\common\public_port.py", line 93, in judge_method
    resp=self.post(url=url,data=data,headers=headers)
  File "C:\woniuboss\common\public_port.py", line 56, in post
    return self.session.post(url,data,headers=headers,file=file)
  File "C:\python3.6\lib\site-packages\requests\sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_student_mode.py", line 243, in test_stu_08
    self.student(moduname)
  File "C:\woniuboss\testcases\test_student_mode.py", line 116, in student
    % (code, title,err_msg,str(e)))
UnboundLocalError: local variable 'err_msg' referenced before assignment


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='failClass'>
    <td>test_training_resources_enquiries.decode</td>
    <td>1</td>
    <td>0</td>
    <td>1</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c6',1)">Detail</a></td>
</tr>

<tr id='ft6.1' class='none'>
    <td class='failCase'><div class='testcase'>test_training_resources_enquiries</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.1')" >
        fail</a>

    <div id='div_ft6.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft6.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_enquiries.py", line 36, in test_training_resources_enquiries
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_enquiries.py", line 44, in test_training_resources_enquiries
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_enquiries.py", line 36, in test_training_resources_enquiries
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_enquiries.py", line 44, in test_training_resources_enquiries
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='failClass'>
    <td>test_training_resources_new.decode</td>
    <td>1</td>
    <td>0</td>
    <td>1</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c7',1)">Detail</a></td>
</tr>

<tr id='ft7.1' class='none'>
    <td class='failCase'><div class='testcase'>test_training_resources_new</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.1')" >
        fail</a>

    <div id='div_ft7.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft7.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_new.py", line 37, in test_training_resources_new
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_new.py", line 45, in test_training_resources_new
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_new.py", line 37, in test_training_resources_new
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_new.py", line 45, in test_training_resources_new
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='failClass'>
    <td>test_training_resources_revision.decode</td>
    <td>1</td>
    <td>0</td>
    <td>1</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c8',1)">Detail</a></td>
</tr>

<tr id='ft8.1' class='none'>
    <td class='failCase'><div class='testcase'>test_training_resources_revision</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft8.1')" >
        fail</a>

    <div id='div_ft8.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft8.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft8.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_revision.py", line 40, in test_training_resources_revision
    self.assertEqual(returned_value.text,expection)
AssertionError: '修改成功.' != '修改失败.'
- 修改成功.
+ 修改失败.


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_revision.py", line 44, in test_training_resources_revision
    raise AssertionError(errors)
AssertionError: ["explor-02错误原因：'修改成功.' != '修改失败.'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_revision.py", line 40, in test_training_resources_revision
    self.assertEqual(returned_value.text,expection)
AssertionError: '修改成功.' != '修改失败.'
- 修改成功.
+ 修改失败.


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_revision.py", line 44, in test_training_resources_revision
    raise AssertionError(errors)
AssertionError: ["explor-02错误原因：'修改成功.' != '修改失败.'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='failClass'>
    <td>test_training_resources_scrap.decode</td>
    <td>1</td>
    <td>0</td>
    <td>1</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c9',1)">Detail</a></td>
</tr>

<tr id='ft9.1' class='none'>
    <td class='failCase'><div class='testcase'>test_training_resources_scrap</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.1')" >
        fail</a>

    <div id='div_ft9.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft9.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_scrap.py", line 37, in test_training_resources_scrap
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_scrap.py", line 45, in test_training_resources_scrap
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_scrap.py", line 37, in test_training_resources_scrap
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_training_resources_scrap.py", line 45, in test_training_resources_scrap
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='failClass'>
    <td>test_transfer_to_responsible_person.decode</td>
    <td>1</td>
    <td>0</td>
    <td>1</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c10',1)">Detail</a></td>
</tr>

<tr id='ft10.1' class='none'>
    <td class='failCase'><div class='testcase'>test_transfer_to_responsible_person</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft10.1')" >
        fail</a>

    <div id='div_ft10.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft10.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft10.1: Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_transfer_to_responsible_person.py", line 37, in test_transfer_to_responsible_person
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_transfer_to_responsible_person.py", line 43, in test_transfer_to_responsible_person
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]
Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_transfer_to_responsible_person.py", line 37, in test_transfer_to_responsible_person
    returned_value=self.WoniuBoss.judge_method(method=method,url=url,data=parameter,headers=None)
TypeError: request() got an unexpected keyword argument 'file'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\woniuboss\testcases\test_transfer_to_responsible_person.py", line 43, in test_transfer_to_responsible_person
    raise AssertionError(errors)
AssertionError: ["explor-01错误原因：request() got an unexpected keyword argument 'file'"]


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='total_row'>
    <td>Total</td>
    <td>37</td>
    <td>1</td>
    <td>10</td>
    <td>26</td>
    <td>&nbsp;</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>

</body>
</html>
